### DISCUSSION TITLE
Welcome to FABM Discussions!

### DISCUSSION THREAD

**Discussion by jornbr** [fabm-model/fabm#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with the FABM community. It is a forum for both users and developers, so feel free to ask about use of FABM (for instance, `fabm.yaml`), developing new biogeochemical models, developing couplers for hydrodynamic models, and anything else FABM related. You can also use this forum to announce the availability of new FABM-based biogeochemical models or FABM-supporting hydrodynamic models.

  You will need to be logged into GitHub to start and contribute to discussions.

  If you want to be notified via e-mail when new items are posted to discussions, click the arrow next to the "Watch" button that you can find in [the Code section](https://github.com/fabm-model/fabm). Then choose what activity you would like to be notified about, for instance, Custom -> Discussions. 

  These discussions replace the [`fabm-users`](https://groups.google.com/g/fabm-users) and [`fabm-devel`](https://groups.google.com/g/fabm-devel) mailing lists. These are no longer active, but you may want to revisit them to see past questions and answers.

  If you would like to report bugs in FABM or request new features, you could consider submitting these directly as [issue](../issues).

**Comment by adolfstips** [comment]
Dear All,

our modelling group at JRC (Ispra, Italy is looking for a biogeochemical modeller,
working as a Contract Agent on a 3 year project, shedding light on European blue carbon.
 
https://recruitment.jrc.ec.europa.eu/showprj_iframe.php?id=2759&apply=vacancy-for-auxiliary.php?inst=200125SDP&pid=2759&target=_blank#toolbar=0

Cheers, Adolf
